src/clients/openapi.js

```diff
@@ -49,6 +49,10 @@ function defaultExtendHeaders(req, headers) {
         extendHeaders['X-Request-User'] = userId;
     }
 
+    // pass the jwt
+    const jwt = get(req, 'locals.jwt');
+    extendHeaders.jwt = JSON.stringify(jwt);
```

Comment:
what format is jwt here if it's not already a string?

Reply:
It'd be more standard to send this as `Authorization: Bearer <token>`

Reply:
Well it actually looks like it's been digested, it's an object, e.g.:

```
{"email":"stuart.hadfield+test@globality.company","email_verified":false,"app_metadata":{},"iss":"https://auth0.dev.globality.io/","sub":"auth0|5f184b074928f60019f56947","aud":"1gG3HAbiBXMiKpnkwot8QDE4HJK1DRW0","iat":1615369332,"exp":1615405332}
```

Reply:
right. if this is plain text, it doesn't really increase our confidence on the BE about the provenance of the request. Is the original token still kicking around somewhere?

Reply:
There's nothing really super obvious. The bearer token is probably stored in a session cookie which we could probably find.